styles/global.ts

```diff
@@ -0,0 +1,50 @@
+import { createGlobalStyle } from 'styled-components';
+
+export const GlobalStyles = createGlobalStyle`
+  html {
+    box-sizing: border-box;
+  }
+
+  *,
+  *::before,
+  *::after {
+    box-sizing: inherit;
+  }
+
+  body,
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6,
+  ul,
+  ol,
+  li,
+  p,
+  pre,
+  blockquote,
+  figure,
+  hr {
+    margin: 0;
+    padding: 0;
+  }
+
+  input,
+  textarea,
+  select,
+  button {
+    color: inherit;
+    font: inherit;
+    letter-spacing: inherit;
+  }
+
+  embed,
+  iframe,
+  img,
+  object,
+  video {
```

Comment:
Also, add the following tags:

```html	
article,
aside,
details,
figcaption,
figure,
footer,
header,
hgroup,
nav,
section,
summary
```

Reply:
what do you mean by adding these elements? Do they need some reset in some form?

Reply:
not sure about IE6/7/8/9. I don't think we should bother supporting those browsers

Reply:
We have a browser support conversation on #144.

Reply:
> what do you mean by adding these elements? Do they need some reset in some form?

Yes, I meant adding reset rules for the mentioned tags

Reply:
That's not necessary